account_banking_payment_export/wizard/payment_order_create.py

```diff
@@ -83,7 +80,7 @@ def filter_lines(self, lines):
         """
         self.ensure_one()
         payment_lines = self.env['payment.line'].\
-            search([('order_id.state', 'in', ('draft', 'open')),
+            search([('order_id.state', '!=', 'cancel'),
                     ('move_line_id', 'in', lines.ids)])
```

Comment:
Moving payment lines out of a sent payment order does not look right to me, as then the payment order would not reflect what has been sent to the bank.

@ecino I think there is a state on payment.order.line too, perhaps it can be used in the filter? 

@alexis-via you may be interested in this conversation.